src/main/java/com/bitso/Bitso.java

```diff
@@ -174,6 +178,41 @@ public BitsoOpenOrders getOpenOrders(BitsoBook book) {
     }
 
     /**
+     * @param The
+     *            specific endpoint to get from a {@link com.bitso.exchange.BitsoLedgerEntry}.
+     * @return The {@link com.bitso.exchange.BitsoLedgerEntry} of the user.
+     */
+    public BitsoLedgerEntry getLedger(String endpoint) {
```

Comment:
Sería mejor crear un enum y pasar un enum en vez de pasar un String. Much cleaner use of java.